test/test_t2ramsey.py

```diff
@@ -43,7 +43,7 @@ def __init__(
         configuration = QasmBackendConfiguration(
             backend_name="T2Ramsey_simulator",
             backend_version="0",
-            n_qubits=int(1e6),
+            n_qubits=100,
```

Comment:
Do we still have two duplicates of the T2Ramsey backend ?!

Reply:
Yeah, but seems like not a scope of this PR.